iidm/iidm-impl/src/test/java/com/powsybl/iidm/network/impl/BusBreakerVoltageLevelTest.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.math.graph.*;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Florian Dupuy <florian.dupuy at rte-france.com>
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class BusBreakerVoltageLevelTest {
```

Comment:
This test is copied from the powsybl-tutorials repo. I couldn't find existing BusBreakerVoltageLevel tests, so I added this file.